### PR TITLE
realtek-mst: Add a warning that extreme caution should be used

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [X] Documentation

### CODE REVIEW

File: plugins/realtek-mst/README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 This plugin updates the firmware of DisplayPort MST hub devices made by Realtek,
 such as the RTD2141b and RTD2142.
 
+This plugin was neither written by, verified or endorsed by Realtek Semiconductor
+Corp, and so should be used with extreme caution.
```
Comment (superm1) [inline plugins/realtek-mst/README.md:9]: how is this any different than most of the other plugins?  Why is it special to receive such a message in README?

Comment (hughsie) [reply plugins/realtek-mst/README.md:9]: Realtek expressed concern.

Comment (superm1) [reply plugins/realtek-mst/README.md:9]: So instead, can't they review the plugin and endorse it?  It feels like it's 2 steps away from person reports "there is a problem with that tool" to realtek and they say "we only support `realtek-mst-flasher.so`".

Comment (superm1) [reply plugins/realtek-mst/README.md:9]: Specifically r8152 comes to mind...

Comment (hughsie) [reply plugins/realtek-mst/README.md:9]: I'll poke them offline.

Comment (tari) [reply plugins/realtek-mst/README.md:9]: The implication that the correctness of the plugin ("extreme caution") depends on Realtek's endorsement seems weird- what about "it is not supported by Realtek," which would also short-circuit @superm1's hypothetical?

Comment (hughsie) [reply plugins/realtek-mst/README.md:9]: So `This plugin was neither written by, verified, supported or endorsed by Realtek Semiconductor Corp.`? Or still too wordy?

Comment (tari) [reply plugins/realtek-mst/README.md:9]: Could remove the first "by", but I'm happy with that.

Comment (hughsie) [reply plugins/realtek-mst/README.md]: > So instead, can't they review the plugin and endorse it

That'd be a _no_ from them.

**Review by tari — APPROVED** [review]
The commit summary might be edited since the README no longer recommends extreme caution, but content changes LGTM.

**Comment by Realtek-William** [comment]
We saw that the discussion in Github will only add a warning documentation file.
We hope Google can use another word to replace “realtek”  word of the plugin’s name, instead of only adding a documentation to illustrate.
